server/controllers/payment.js

```diff
@@ -0,0 +1,123 @@
+const mongoose = require("mongoose");
+const stripe = require("stripe")(process.env.STRIPE_KEY);
+const User = require("../models/User");
+const Profile = require("../Models/Profile");
+const asyncHandler = require("express-async-handler");
+const verifyProfile = require("../utils/verifyProfile");
+const verifyStripeCustomer = require("../utils/verifyStripeCustomer");
+
+// @route POST /payment
+// @desc Create new customer
+// @access Private
+exports.createCustomer = asyncHandler(async (req, res, next) => {
+  const profile = await verifyProfile(req);
+
+  const customer = await stripe.customers.create();
```

Comment:
Optional: Maybe we can also perform a check if a user already has a stripe customer id before creating one? 😄